addroom.html

```diff
@@ -0,0 +1,185 @@
+            <div class="form-group">
+              <label for="code">
+                Postal code
+              </label>
+              <input type="text" class="form-control" id="code" name="code" />
+            </div>
+            <button type="submit" class="btn btn-primary">
+              Create your account
+            </button>
+          </form>
+        </div>
+      </div>
+    </div>
+    <script>
+      var map;
+      var lat;
+      var lng;
+      var placeid;
+      function initMap() {
+        map = new google.maps.Map(document.getElementById("map"), {
+          center: { lat: -34.397, lng: 150.644 },
+          zoom: 8
+        });
+      }
+      function remap() {
+        var geocoder = new google.maps.Geocoder();
+        geocodeAddress(geocoder, map);
+        function geocodeAddress(geocoder, resultsMap) {
+          var street = document.getElementById("street").value;
+          var city = document.getElementById("City").value;
+          var provinces = document.getElementById("Provinces").value;
+          var address = street + ", " + city + ", " + provinces;
+
+          geocoder.geocode({ address: address }, function(results, status) {
+            if (status === "OK") {
+              resultsMap.setCenter(results[0].geometry.location);
+              map.setZoom(20);
+              var marker = new google.maps.Marker({
+                map: resultsMap,
+                position: results[0].geometry.location
+              });
+              lat = results[0].geometry.location.lat();
+              lng = results[0].geometry.location.lng();
+              placeid = results[0].place_id;
+            } else {
+              alert(
+                "Geocode was not successful for the following reason: " + status
+              );
+            }
+          });
+        }
+      }
+      var form_login = document.getElementById("add-form");
+      form_login.onsubmit = function(e) {
+        e.preventDefault();
+        var housename = document.getElementById("name").value;
+        var streetname = document.getElementById("street").value;
+        var cityname = document.getElementById("City").value;
+        var provincename = document.getElementById("Provinces").value;
+        var postalcode = document.getElementById("code").value;
+        $.ajax({
+          type: "POST",
+          url: "http://192.168.1.104:8000/house/create",
+          xhrFields: {
+            withCredentials: true 
+          },
+          crossDomain: true,
+          data: {
+            house_name: housename,
+            place_id: placeid,
+            house_address: streetname,
+            house_city: cityname,
+            house_province: provincename,
+            house_postcode: postalcode,
+            longitude: lng,
+            latitude: lat
+          },
+          success: function(msg) {
+            console.log("hello");
+          },
+          fail: function() {
+            console.log("add failed");
```

Comment:
Same here, try to make log info more specific.

Reply:
I have fixed it. Please check